src/plugins/intel_cpu/src/transformations/snippets/x64/pass/brgemm_to_brgemm_cpu.cpp

```diff
@@ -21,6 +22,21 @@
 
 namespace ov {
 namespace intel_cpu {
+namespace {
+inline std::vector<size_t> make_subtensor(const ov::Shape& tensor) {
+    return std::vector<size_t>(std::min(tensor.size(), 2lu), ngraph::snippets::PortDescriptor::Scheduling::FULL_DIM);
+}
+template<typename T>
+void set_full_port_desc(const T& port) {
+    const auto& shape = port.get_shape();
+    ngraph::snippets::PortManager::set_port_descriptor_ptr(port, std::make_shared<ngraph::snippets::PortDescriptor>(shape,
```

Comment:
Do we absolutely need subtensor filled with `FULL_DIM` here? Can it just be equal to shape?

Reply:
TODO: Discussed offline: use `VECTOR_SIZE` instead of `FULL_DIM`